onnxruntime/core/providers/webnn/builders/model_builder.cc

```diff
@@ -123,39 +123,37 @@ Status ModelBuilder::RegisterInitializers() {
     emscripten::val operand = emscripten::val::object();
     if (IsSupportedDataType(data_type, wnn_device_type_)) {
       ORT_RETURN_IF_NOT(SetWebnnDataType(desc, data_type), "Unsupported data type");
-      unpacked_tensors_.push_back({});
-      std::vector<uint8_t>& unpacked_tensor = unpacked_tensors_.back();
-      ORT_RETURN_IF_ERROR(onnxruntime::utils::UnpackInitializerData(tensor, unpacked_tensor));
+      auto * tensorPtr = (std::byte*)(tensor.raw_data().c_str());
```

Comment:
Does this always work, even for cases where the `TensorProto` uses zig zag encoding for `int64_data` or `int32_data` fields? Even though every model I see nowadays just uses the `TensorProto::raw_data` field, it's still possible (unlikely but possible) that some tensors will need unpacking. So maybe this needs to be a conditional check to call `onnxruntime::utils::UnpackInitializerData` in those cases.

Reply:
[nit] `auto* tensorPtr`

Reply:
`reinterpret_cast<std::byte*>`

Reply:
Fixed, PTAL again, thanks!